share/jupyterhub/templates/page.html

```diff
@@ -68,7 +68,16 @@ <h1 class="modal-title" id="{{key}}-label">{{title}}</h1>
         base_url: "{{base_url}}",
         prefix: "{{prefix}}",
         {% if user %}
-        user: "{{user.json_escaped_name}}",
```

Comment:
I couldn't decide whether `{{user.json_escaped_name|safe}}` is safe

Reply:
`json_escaped_name | safe` ought to be correct, because the escaped value cannot close the quote (arguably json_escaped_name should have _included_ the outer quotes, but it's okay).